### PR TITLE
Enable ARM64 build environment

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
As powerful arm64 laptops enter the market, it may be useful to build LmP and other yocto variants on arm64.

Here are the changes needed to support building on arm64, along with x86-64.

Wasn't sure if we needed - https://github.com/foundriesio/lmp-manifest/commit/b4f87e26330a8f34852708c7b3eab01d85d32fcf anymore, and I've moved this to `libc6-dev`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,10 +3,15 @@ FROM ubuntu:20.04 AS container-tools
...
+        file
 
-RUN wget -P /tmp https://go.dev/dl/go1.18.linux-amd64.tar.gz && \
-	tar -C /usr/local -xzf /tmp/go1.18.linux-amd64.tar.gz
+# Detect host architecture
+RUN file /bin/bash | grep -q x86-64 && echo amd64 > /tmp/arch || true
+RUN file /bin/bash | grep -q aarch64 && echo arm64 > /tmp/arch || true
+
+RUN wget -P /tmp https://go.dev/dl/go1.18.linux-$(cat /tmp/arch).tar.gz && \
+	tar -C /usr/local -xzf /tmp/go1.18.linux-$(cat /tmp/arch).tar.gz
```
Comment (doanac) [inline Dockerfile:14]: mind blown

**Review by doanac — COMMENTED** [review]
i'm all for this. we might have to change the CI flow to build a multi-arch container as well

**Comment by ricardosalveti** [comment]
+1 for the change, mind breaking it in two commits? One for removing libc6-dev-i386 and the other to update the go-lang logic?

File: Dockerfile [hunk 2]
```diff
@@ -17,9 +22,14 @@ RUN git clone https://github.com/containers/skopeo.git /skopeo && \
 # Build ostreeuploader, aka fiopush/fiocheck
 FROM ubuntu:20.04 AS fiotools
 RUN apt-get update
-RUN apt-get install -y wget git gcc make -y
-RUN wget -P /tmp https://go.dev/dl/go1.19.9.linux-amd64.tar.gz && \
-    tar -C /usr/local -xzf /tmp/go1.19.9.linux-amd64.tar.gz
+RUN apt-get install -y wget git gcc make file
+
+# Detect host architecture
```
Comment (MrCry0) [inline Dockerfile:27]: Do we need to detect arch twice - here and at #9?

File: Dockerfile (hunk 1 above)
Comment (ricardosalveti) [reply Dockerfile]: @EmbeddedAndroid it seems this got removed (the wget line) in your latest update?

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**